### DISCUSSION TITLE
Tags

### DISCUSSION THREAD

**Discussion by Courtny** [microsoft/atlas-design#171] [Component]
Tags allow us to label, organize, and discover related content.

**Comment by jdanyow** [comment]
@Courtny @wibjorn thoughts on introducing a new component for "states" and discontinuing the usage of large tags? To my eye the large tag looks a lot like a button.

https://review.docs.microsoft.com/en-us/test/docs/known-issue/?branch=main
![image](https://user-images.githubusercontent.com/1306747/136843233-fc75b6fb-1dfb-46a5-9d01-3360ae2abbee.png)

Borrowing from the primer [State component](https://primer.style/css/components/labels#states)
![image](https://user-images.githubusercontent.com/1306747/136843329-33c802a4-5b45-47a1-a69f-e59640404aa0.png)
